api/src/test/java/org/openmrs/BaseFormRecordableOpenmrsDataTest.java

```diff
@@ -37,7 +37,7 @@ public void setFormField_shouldSetTheUnderlyingFormNamespaceAndPathInTheCorrectP
 		impl.setFormField(ns, path);
 		Field formNamespaceAndPathProperty = BaseFormRecordableOpenmrsData.class.getDeclaredField("formNamespaceAndPath");
 		formNamespaceAndPathProperty.setAccessible(true);
-		Assert.assertEquals(ns + FORM_NAMESPACE_PATH_SEPARATOR + path, formNamespaceAndPathProperty.get(impl));
+		Assertions.assertEquals(ns + FORM_NAMESPACE_PATH_SEPARATOR + path, formNamespaceAndPathProperty.get(impl));
```

Comment:
we generally favor static imports of assertion methods over the use of `Assertions.yyy` but other than that it looks good from JUnit 5 side :+1:

Reply:
@icrc-loliveira what @teleivo suggests is to do:
```java
import static org.junit.jupiter.api.Assertions.assertEquals;

...

assertEquals(ns + FORM_NAMESPACE_PATH_SEPARATOR + path, formNamespaceAndPathProperty.get(impl));
```
instead of
```java
import org.junit.jupiter.api.Assertions;

...

Assertions.assertEquals(ns + FORM_NAMESPACE_PATH_SEPARATOR + path, formNamespaceAndPathProperty.get(impl));
```